### ISSUE TITLE
Can't read V7 BYML files because of unknown type

### ISSUE THREAD

**Issue by samidalil** [leoetlino/byml-v2#19]
Hi,

When trying to read a BYML file from bytes with a Python script, the following error occurs :
![image](https://github.com/zeldamods/byml-v2/assets/46566751/c4896a84-0b3e-44ba-9c8c-b2d069d1b445)

As V7, the BYML format now includes the "container" node type which identifier is "0x20".

If the lib has not been updated, do you think I could add the new types through a PR ?

**Comment by leoetlino** [comment]
Yes, v7 nodes are not supported yet. I'll add support soon to byml-v2 and oead soon
